source/_components/switch.xiaomi_vacuum

```diff
@@ -0,0 +1,36 @@
+The `xiaomi_vacuum`switch platform allows you to control the state of your [Xiaomi Mi Robot Vacuum](http://www.mi.com/roomrobot/). 
+Current supported features are start and stop (go to dock).
+
+{% linkable_title Getting started %}
+
+To be able to communicate with the vacuum one needs to have its IP address as well as an encryption token.
+This token is only attainable before the device has been connected over the app to your local wifi.
+
+In order to fetch the token, reset the robot, connect to its the network its announcing (rockrobo-XXXX)
+and run the following command:
+
+```
+mirobo discover
+```
+
+which should return something similar to this:
+
+```
+INFO:mirobo.vacuum:  IP 192.168.8.1: Xiaomi Mi Robot Vacuum - token: b'ffffffffffffffffffffffffffffffff'
+```
+
+If the value is as shown above, full of ffs, the vacuum has already been connected and it needs a reset.
```

Comment:
"full of ffs" sounds weird. Maybe it'd be better to rewrite it in a more user-friendly manner?